Sources/ChessKit/Position.swift

```diff
@@ -29,6 +29,16 @@ public struct Position: Equatable, Sendable {
     /// en passant.
     var enPassant: EnPassant?
 
+    var hashed: Int {
+        var hasher: Hasher = .init()
+        
+        hasher.combine(pieceSet)
+        hasher.combine(sideToMove)
+        hasher.combine(legalCastlings)
+        hasher.combine(enPassant)
+        
+        return hasher.finalize()
+    }
```

Comment:
You can remove this property and just conform `Position` to `Hashable`, it will then have a `hashValue` property which will do this automatically